controller/genre.js

```diff
@@ -1,30 +1,15 @@
-const mongoose = require("mongoose");
-const Genre = require("../models/genre");
-
-exports.GET_ALL_GENRES = (req, res) => {
-  Genre.find()
-    .then((docs) => {
-      return res.status(200).json(docs);
-    })
-    .catch((err) => res.status(500).json(err));
-};
-
-exports.ADD_GENRE = (req, res) => {
```

Comment:
Can you please keep the adding genre functionality? Is there a reason behind removing it? 

I see that it's not [exported](https://github.com/orifmilod/iCinema/blob/25af3b371166fc352b4bdda3cb1517e6713a180a/routes/genres.js) right now? or perhaps we can just add it or at least keep it the same way as it is right now

Reply:
I was developing a functionality that automatically assigns a genre when adding a movie. With this approach, every genre will always have at least one movie associated with it.

Reply:
Not yet! but I'll include it ASAP

Reply:
That's fine. Do you want to include it in this PR or do you want to create a separate one?

Reply:
I'll create a separate PR that includes the absolute path instead of the relative path, along with the genre functionality. If you have any additional suggestions or ideas for this project, please let me know. I'm open to incorporating them into the codebase."

Reply:
Sounds good. 
Honestly, I don't know what other feature would be nice to be added to the project off the top of my head. Pleas feel free to add anything that you find useless. I'd be happy to merge them.